go.mod

```diff
@@ -14,7 +14,8 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.0
 	github.com/test-network-function/graphsolver-lib v0.0.0-20220923161551-922faf8697be
```

Comment:
This should be upgraded to
v0.0.0-20220926190927-5f72a82c96d4 => v0.0.0-20220929081721-4e9a42510716
or else it error's out
ptpInterface.IfPci.Description undefined

Reply:
fixed